setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="ccnmtlsettings",
-    version="1.1.3",
+    version="1.1.2",
```

Comment:
semver forbids modifying existing releases (since it's already out on pypi, in pip caches on any machine that built it, etc). It will need to be rolled forward rather than back.

I also strongly request that people don't bump version numbers in the same commit/PR as changes to the code.